nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-canvas-utils.js

```diff
@@ -412,9 +412,9 @@
 
                 // create object whose keys are the parameter name and the values are the parameter values
                 var paramsObject = {};
-                params.forEach(function (v, k) {
-                    paramsObject[k] = v;
-                });
+                for(var pair of params.entries()) {
```

Comment:
Interesting, I see what you mean.  When I build the nifi-web-ui module with the "of" in place, I get the following error:
[ERROR] /var/local/workspace/nifi/nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-canvas-utils.js [1:0]: Compilation produced 301 syntax errors.
[ERROR] ...nf-canvas-utils.js:line 1:column 0:Compilation produced 301 syntax errors.

However, when I change "of " to "in", the error goes away and it is able to build the nar.  How did you get around this issue?